Header header
geometry_msgs/PoseStamped pose
training_q_learning/TwistStampedModified twist
geometry_msgs/Vector3Stamped linear_acceleration
